config/filter.d/exim.conf

```diff
@@ -18,7 +18,7 @@ failregex = ^%(pid)s %(host_info)ssender verify fail for <\S+>: (?:Unknown user|
             ^%(pid)s %(host_info)srejected RCPT [^@]+@\S+: (?:relay not permitted|Sender verify failed|Unknown user|Unrouteable address)\s*$
             ^%(pid)s SMTP protocol synchronization error \([^)]*\): rejected (?:connection from|"\S+") %(host_info)s(?:next )?input=".*"\s*$
             ^%(pid)s SMTP call from \S+ %(host_info)sdropped: too many nonmail commands \(last was "\S+"\)\s*$
-            ^%(pid)s SMTP protocol error in "[^"]+" %(host_info)sAUTH command used when not advertised\s*$
+            ^%(pid)s SMTP protocol error in "[^"]+(?:"+[^"]*(?="))*?" %(host_info)sAUTH command used when not advertised\s*$
```

Comment:
This is the first time I meet lookarounds and I also do not have much experience with greedy/non-greedy, so this is a bit difficult for me to grasp. It seems the `(?=")` is there to make sure that there is always at least one `"` left later in the string while the engine keeps matching occurrences of `"+[^"]*`. Just some comments:
- I do not have experience with regex performance, but I can imagine there is some benefit in this 'narrower' approach over a catch-all.
- I did wonder if things could go wrong if `" [victimIP]` was injected in a later foreign input field. However, I checked that Exim does not accept `"` in `EHLO` and assume that `I=` (interface) is not foreign input and `U=` (user) will not be filled because we have not passed `AUTH`.
- Tests results are ok.